core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -161,6 +161,10 @@
   /** CONV function converts the given number n from one base to another base. */
   public static final SqlFunction CONV = SqlLibraryOperators.CONV;
 
+  /** Returns the index of search string in source string
+   *  0 is returned when no match is found. */
+  public static final SqlFunction INSTR = SqlLibraryOperators.INSTR;
```

Comment:
we are not allowed to modify this class. remove this

Reply:
done